data/net/minecraft/advancements/critereon/DistanceTrigger.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS net/minecraft/advancements/critereon/DistanceTrigger
+	METHOD <init> (Lnet/minecraft/resources/ResourceLocation;)V
+		ARG 1 id
+	METHOD lambda$trigger$0 (Lnet/minecraft/server/level/ServerPlayer;Lnet/minecraft/world/phys/Vec3;Lnet/minecraft/world/phys/Vec3;Lnet/minecraft/advancements/critereon/DistanceTrigger$TriggerInstance;)Z
+		ARG 3 instance
+	METHOD trigger (Lnet/minecraft/server/level/ServerPlayer;Lnet/minecraft/world/phys/Vec3;)V
+		ARG 1 player
+		ARG 2 position
+	CLASS TriggerInstance
+		METHOD <init> (Lnet/minecraft/resources/ResourceLocation;Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;Lnet/minecraft/advancements/critereon/LocationPredicate;Lnet/minecraft/advancements/critereon/DistancePredicate;)V
+			ARG 1 id
+			ARG 2 player
+			ARG 3 startPosition
+			ARG 4 distance
+		METHOD fallFromHeight (Lnet/minecraft/advancements/critereon/EntityPredicate$Builder;Lnet/minecraft/advancements/critereon/DistancePredicate;Lnet/minecraft/advancements/critereon/LocationPredicate;)Lnet/minecraft/advancements/critereon/DistanceTrigger$TriggerInstance;
+			ARG 0 builder
+			ARG 1 distancePredicate
+			ARG 2 startPosition
+		METHOD matches (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/world/phys/Vec3;Lnet/minecraft/world/phys/Vec3;)Z
+			ARG 1 level
+			ARG 2 here
+			ARG 3 there
```

Comment:
These should either be documented or be given more descriptive names (possibly `startPosition` and `currentPosition`?)